build_variables.bzl

```diff
@@ -841,6 +841,7 @@ libtorch_python_core_sources = [
     "torch/csrc/dynamo/cpp_shim.cpp",
     "torch/csrc/dynamo/cpython_defs.c",
     "torch/csrc/dynamo/eval_frame.c",
+    "torch/csrc/dynamo/eval_frame.cpp",
```

Comment:
Won't two files with the same base name cause build issues in some places?  (Both try to write `eval_frame.o`.)  It might be safer to pick a different name.  They should also have different header files for clarity of what comes from where.